### ISSUE TITLE
JetElements. Scroll navigation idea

### ISSUE THREAD

**Issue by aksamitnaya** [CrocoBlock/suggestions#342]
I would like to use the scroll navigation widget as the position is fixed on the side and it is built to be used as a navigation on the page. Maybe crocoblock could add this feature as the scroll navigation widget has not got a good usability. People usually hover on the bullet point and then hover on the tip. But then the tip is gone and you cannot click on the link, so you need to hover again on the bullet point and click on the bullet point. This is complicated and makes the scroll navigation widget useless in my opinion, as it makes the user thinking on how to use the navigation.

**Comment by mrusteberg** [comment]
Yes, I very much agree with your suggestion. I would also suggest to modify the scroll navigation into an overall on-page navigation feature, something that's really missing in JetElements despite Crocoblock using that feature extensively on their own site.
